resources/sql/application-queries.sql

```diff
@@ -1,15 +1,16 @@
 -- name: yesql-add-application-query<!
 -- Add application
 insert into applications
-(form_id, key, content, lang, preferred_name, last_name, hakukohde, hakukohde_name)
+(form_id, key, content, lang, preferred_name, last_name, hakukohde, hakukohde_name, secret)
 values
-(:form_id, :key, :content, :lang, :preferred_name, :last_name, :hakukohde, :hakukohde_name);
+(:form_id, :key, :content, :lang, :preferred_name, :last_name, :hakukohde, :hakukohde_name, :secret);
 
 -- name: yesql-get-application-list
 select a.id,
   a.key, a.lang,
   a.preferred_name || ' ' ||  a.last_name as applicant_name,
-  a.created_time, coalesce(ar.state, 'received') as state
+  a.created_time, coalesce(ar.state, 'received') as state,
+  a.secret
```

Comment:
Do we actually need to return the secret in all these queries? I'm worried that it will leak by accident via some REST API calls or such.